test/integration/next/package.json

```diff
@@ -5,7 +5,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "test": "playwright test",
+    "test": "npm run test:types && playwright test",
+    "test:types": "npx tsd --files tests/types/*.test-d.ts",
```

Comment:
introduce basics typing tests for the `@supabase/ssr` context. Since it depends of `supabase-js` as peer deps, it'll help detect uncaught breaking change that would break compatibility between `ssr` and `supabase-js` previous verison.